gridData/core.py

```diff
@@ -52,74 +52,140 @@ def _grid(x):
 
 
 class Grid(object):
```

Comment:
grid.core shows up after formats in the list, would it make more sense to order it beforehand? (i.e. it seems like something that folks would want to know before they started dealing with different formats [if that makes sense]).

Reply:
changed order